easybuild/framework/easyconfig/style.py

```diff
@@ -62,27 +70,25 @@ def _eb_check_trailing_whitespace(physical_line, lines, line_number, total_lines
     The arguments are explained at
     https://pep8.readthedocs.org/en/latest/developer.html#contribute
     """
-    comment_re = re.compile(r'^\s*#')
-    if comment_re.match(physical_line):
+    # apparently this is not the same as physical_line line?!
+    line = lines[line_number-1]
```

Comment:
Because apparently this is not the same as `physical_line`... I noticed that for the `description =` line, only the part *after* the `=` was included in `physical_line`...